### PR TITLE
docs(css): FRMW-290-Document 120 char line limit

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-290 

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Update CSS Styleguide to establish 120 character line limit, which is inline with Javascript Styleguide.
> * Line length should be under 120 characters

### CODE REVIEW

**Comment by bry4n** [comment]
LGTM

**Comment by glynnis** [comment]
LGTM

File: guides/css-styleguide.md [hunk 1]
```diff
@@ -2,23 +2,30 @@
...
 
     *Scenario #1*
 
     Use `.blog-article {}` instead of `article {}`.
 
-    Yes, sometimes it's "uglier" to use `blog-article` as a class, but it's much more re-usable than relying on the specific HTML `article` element.
-    Along that same logic, [don't style heading elements](http://www.stubbornella.org/content/2011/09/06/style-headings-using-html5-sections/),
+    Yes, sometimes it's "uglier" to use `blog-article` as a class, but it's much more re-usable than relying on the 
+    specific HTML `article` element.
+    Along that same logic,
```
Comment (CITguy) [inline guides/css-styleguide.md]: If you're addressing line length and combining, I'd combine this line with the one above.

File: guides/css-styleguide.md [hunk 2]
```diff
@@ -46,20 +57,27 @@ Most of these standards are based off of the guidelines provided by [SMACSS](htt
...
 
 ## Format
 
 **Note**: This section inspired by [The Principles of writing consistent, idiomatic CSS](https://github.com/necolas/idiomatic-css/blob/master/README.md#format)
 
-This code format ensures that code is easy to read, easy to clearly comment, minimizes the chance of accidentally introducing errors, and results in useful diffs and blames.
+This code format ensures that code is easy to read, easy to clearly comment, minimizes the chance of accidentally 
+introducing errors, and results in useful diffs and blames.
 
+* Line length should be under 120 characters
```
Comment (CITguy) [inline guides/css-styleguide.md]: I'd say `no more than 120 characters`

**Comment by 100stacks** [comment]
@CITguy comments implemented. :+1:

**Comment by CITguy** [comment]
LGTM

**Pull request merged**